indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -267,6 +267,7 @@
           :short-name.lowercase  m/string-field-mapping
           :version-id            (m/stored m/string-field-mapping)
           :version-id.lowercase  m/string-field-mapping
+          :parsed-version-id.lowercase m/string-field-mapping
```

Comment:
Make sure to note we need to reindex collections as part of the deployment instructions.